BitwardenShared/UI/Platform/Application/Utilities/Coordinator.swift

```diff
@@ -172,10 +174,12 @@ extension Coordinator where Self: HasNavigator {
 
     /// Shows the toast.
     ///
-    /// - Parameter text: The text of the toast to display.
+    /// - Parameters:
+    ///   - title: The title text displayed in the toast.
+    ///   - title: The subtitle text displayed in the toast.
     ///
-    func showToast(_ text: String) {
-        navigator?.showToast(Toast(text: text))
+    func showToast(_ title: String, subtitle: String? = nil) {
+        navigator?.showToast(Toast(title: title, subtitle: subtitle))
```

Comment:
same question here, we can't test this, right?

Reply:
Added a test for this!